scripts/enable-ecs-agent-gpu-support.sh

```diff
@@ -60,6 +59,12 @@ if [[ $AMI_TYPE != "al2keplergpu" && -z ${AIR_GAPPED} ]]; then
     sudo yum remove -y kmod-nvidia-open-dkms
     sudo yum-config-manager --disable cuda-rhel7.repo
     sudo rm /etc/yum.repos.d/cuda-rhel7.repo
+    # epel was used to install dkms, now delete as it points to public http endpoints which
+    # can cause problems for customers using isolated subnets.
+    sudo yum-config-manager --disable epel
+    sudo rm /etc/yum.repos.d/epel.repo
+    sudo rm /etc/yum.repos.d/epel-testing.repo
+    sudo amazon-linux-extras disable epel
```

Comment:
good catch -- I had to go validate this myself to see that the epel topic was auto-enabled by the install.